### PR TITLE
Translate to Russian (complete)

### PR DESCRIPTION
**Pull request by Joes-Madness**
Translated to Russian

### CODE REVIEW

**Comment by kedMertens** [comment]
Great work! 
Could you please edit : "Бот находится в постоянном развитии!"
I believe that development in that case means - " разработка", but you are translating it as evaluation. So I believe that proper translation should be - "Бот находится в постоянной разработке!"

And was worth it to jump over heads in order to gain T-shirt? The issue was marked as WIP why would you do it?
@GabrielRF, @rougeth

File: i18n/bot.ru-ru.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+ru-ru:
+  askfile: Отправь мне файл (до 20MB) или ссылку на файл.
+  askemail: Введи Kindle e-mail.
+  askemail2: Введи email, используемый в аккаунте Amazon.
```
Comment (kedMertens) [inline i18n/bot.ru-ru.yml]: Could you edit "email" as it is done in rest of the file, "e-mail" ?

File: i18n/bot.ru-ru.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  startolduser: |
+    <b>Приветствую</b>!
+    Зарегистрированные e-mail:
+    {} {}
+    {} {}
+    Для отправки файла на Kindle, нажми <b>Отправить файл</b>.
+    Для смены e-mail, нажми <b>Настройка e-mail</b>.
+  error: |
+    <b>Ошибка</b>.
+    Нажми /start и проверь e-mails.
```
Comment (kedMertens) [inline i18n/bot.ru-ru.yml]: I believe that " your e-mails" has to be translated in that case. Try to reconsider that one.

File: i18n/bot.ru-ru.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  donate: |
+    <b>Поддержка!</b>
+    http://grf.xyz/paypal\n\n<b>Отправь новую команду</b>.
+  btn1: Отправить файл
+  btn2: Настройка e-mail
+  btn3: Как есть
+  btn4: Конвертировать
+  help: |
+    <b>Что делает этот бот?</b>
+    Этот бот пересылает файла на твой Kindle как если бы ты отправил их по e-mail.
```
Comment (kedMertens) [inline i18n/bot.ru-ru.yml]: Conjugation mistake of word files. Editing is required to "файлы"

File: i18n/bot.ru-ru.yml [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    <b>Какие файлы поддерживаются?</b>
+    Adobe PDF <code>.pdf</code>\nHTML <code>.htm .html</code>
+    Изображения <code>.jpg .gif .bmp .png</code>
+    Mobi book <code>.mobi</code>
+    Microsoft Word <code>.doc .docx</code>
+    Rich Text Format <code>.rtf</code>
+    Текстовые файлы <code>.txt</code>
+    Архивы <code>.zip .xzip</code>
+    <i>* Конвертация файлов в формат Kindle - экспериментальный сервис Amazon.</i>
+    <b>Есть ли лимиты на размер файлов?</b>
```
Comment (kedMertens) [inline i18n/bot.ru-ru.yml]: Don't you think limits will be closer by meaning to "ограничения"?

**Comment by Joes-Madness** [comment]
DelevoXDG thanks, i`m edit translation in startnevuser as in help

**Comment by GabrielRF** [comment]
Is it ready?

**Comment by kedMertens** [comment]
> Is it ready?

I could look through it one more time in several hours

**Comment by kedMertens** [comment]
> 
> 
> Is it ready?

@GabrielRF, It looks fine! All comments were applied. Only one thing, considering PR #40 @Kindle.com has to be corrected and it needs to be accounted that bot.en-us.yml was changed and that issue should be edited too.

**Comment by Joes-Madness** [comment]
I'm a little confused in commits, look, now is ok?

**Comment by GabrielRF** [comment]
Please, would you consider not changing `bot.en-us.yml`?

**Pull request closed without merging**